File: java/Maximum.java
/*
Problem:
You are given N integers length array by the user. Find the maximum number

*/
import java.util.Scanner;

public class Maximum {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the length of the array: ");
        int length = scanner.nextInt();

        int[] numbers = new int[length];

        System.out.println("Enter the array elements:");
        for (int i = 0; i < length; i++) {
            numbers[i] = scanner.nextInt();
        }

        int maximum = numbers[0];
        for (int i = 1; i < length; i++) {
            if (numbers[i] > maximum) {
                maximum = numbers[i];
            }
        }

        System.out.println("Maximum number: " + maximum);
    }
}

File: python/Maximum.py
"""
Problem:
You are given N integers length array by the user. Find the maximum number

"""
n = int(input())
nums = list(map(int, input().split()))

max_num = nums[0]

for i in range(1, n):
    if nums[i] > max_num:
        max_num = nums[i]

print("Maximum number is:", max_num)
